package.json

```diff
@@ -51,6 +51,7 @@
     "@types/express": "^4.17.13",
     "@types/http-errors": "^1.8.2",
     "@types/i18next-fs-backend": "^1.1.2",
+    "@types/jsonwebtoken": "^8.5.8",
```

Comment:
Ah great shout, I forgot that. We can squash these down with the previous when we're done perhaps?